src/components/Dashboard/dashboard.module.scss

```
@@ -0,0 +1,77 @@
+.dashboard { 
+    border-left: #dad8d8 solid 2px;
+    border-radius: 3px;
+    padding-left: 3%;
+    margin: 1%;
+}
+.dashboardContainer {
+    overflow-y: scroll;
+    overflow-x: hidden;
+    height: 800px;
```

```suggestion
    height: 80rem;
```
in general use rem for sizes, ```1rem = 10px``` in our case since the font-size of the html tag is set to 62.5%

check app.scss to see where that style lives